text-ui-test/input.txt

```diff
@@ -0,0 +1,20 @@
+dab
+list
+todo
+todo yeet
+list
+deadline
+deadline hehe
+deadline haha /by dab
+event /at
+event hoho
+event huhu /at yeet
+list
+done -1
+done 1
+done 4
+done 01
+list
+delete 1
+list
+bye
```

Comment:
I think that the testing is insufficient. Maybe you can add extra tests. 
e.g. for deadline command
deadlineedasfa
deadline awrfawesf
deadline dads /by
deadline efaws /by 
deadline /by 2020-10-22
deadline/by 2020-10-22
deadline dda /by 2020-22
deadline dda /by 2020-10-22